local-flake/flake.nix

```
@@ -58,6 +58,54 @@
           ldflags = ["-s" "-w" "-X main.version=${version}"];
         };
 
+        mbt = nixpkgsFor.${system}.buildGoModule rec {
```

I would prefer a more descriptive name here, for example use the repository name itself

```suggestion
        cloud-mta-build-tool = nixpkgsFor.${system}.buildGoModule rec {
```